server/game/cards/04-MM/SirenHorn.js

```diff
@@ -0,0 +1,24 @@
+const Card = require('../../Card.js');
+
+class SirenHorn extends Card {
+    setupCardAbilities(ability) {
+        this.whileAttached({
+            effect: ability.effects.gainAbility('beforeFight', {
+                effect: 'move 1 amber from {0} to {1}',
+                effectArgs: (context) => context.event.card,
+                gameAction: [
+                    ability.actions.removeAmber((context) => ({
+                        target: context.source
+                    })),
+                    ability.actions.placeAmber((context) => ({
```

Comment:
I think this needs to be a `then` clause, otherwise you will place an amber on the fought creatured even if Siren Horn's parent has no amber.

Reply:
Good catch, I've added a condition which should do the same thing

Reply:
Though thinking about it, that wouldn't cover the case where something prevents the amber moving rather than there just not being any..